Runtime/Helpers/CinemachineInputProvider.cs

```diff
@@ -95,9 +95,10 @@ protected InputAction ResolveForPlayer(int axis, InputActionReference actionRef)
                 m_cachedActions[axis].Enable();
             
             return m_cachedActions[axis];
-            // local static function to wrap the lambda which otherwise causes a tiny gc
-            static InputAction GetFirstMatch(in InputUser user, InputActionReference actionRef) => 
-                user.actions.First(x => x.id == actionRef.action.id);
+            
+            // local function to wrap the lambda which otherwise causes a tiny gc
+            InputAction GetFirstMatch(in InputUser user, InputActionReference aRef) => 
+                user.actions.First(x => x.id == aRef.action.id);
```

Comment:
Note: important to pass parameters instead of capturing them. Capturing causes gc.
Previously, I used static function (C#8+ feature), which does not allow capturing local variables. If I pass in all variables that I use, then we won't capture local variables, thus avoiding gc. 

Tested with profiler.